packages/terra-layout/tests/jest/__snapshots__/LayoutSlidePanel.test.jsx.snap

```diff
@@ -21,6 +21,7 @@ exports[`LayoutSlidePanel should render when small 1`] = `
       isRelativeToContainer={true}
       isScrollable={false}
       onRequestClose={[Function]}
+      zIndex="100"
```

Comment:
These snapshots are also updated in this pull request: https://github.com/cerner/terra-framework/pull/320/files

May the better pull request win.

Reply:
💪